adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -93,13 +95,29 @@ - (void)initWithAppId:(nonnull NSString *)appId
   }
 
   _isInitializing = YES;
-  _isBannerPresenting = NO;
 
   // Disable refresh functionality for all banners
   [[VungleSDK sharedSDK] disableBannerRefresh];
 
+  // Set init options for priority placement
+  NSMutableDictionary *initOptions = [NSMutableDictionary dictionary];
+  if (delegate) {
+    NSString *priorityPlacementID = delegate.desiredPlacement;
+    [initOptions setObject:priorityPlacementID forKey:VungleSDKInitOptionKeyPriorityPlacementID];
+    _prioritizedPlacementID = [priorityPlacementID copy];
+
+    NSInteger priorityPlacementAdSize = 1;
+    if ([delegate respondsToSelector:@selector(bannerAdSize)]) {
+      GADAdSize adSize = [delegate bannerAdSize];
+      if (!GADAdSizeEqualToSize(adSize, kGADAdSizeMediumRectangle)) {
```

Comment:
Please add a comment here about why MREC is a special case.

Reply:
@ericleich  Got it, thanks. :)

Reply:
@ericleich Added, thanks. ^_^